docs/zh/latest/plugins/ldap-auth.md

```diff
@@ -0,0 +1,155 @@
+---
+title: ldap-auth
+keywords:
+  - APISIX
+  - Plugin
+  - LDAP Authentication
+  - ldap-auth
+description: 本篇文档介绍了 Apache APISIX ldap-auth 插件的相关信息。
+---
+
+<!--
+#
+# Licensed to the Apache Software Foundation (ASF) under one or more
+# contributor license agreements.  See the NOTICE file distributed with
+# this work for additional information regarding copyright ownership.
+# The ASF licenses this file to You under the Apache License, Version 2.0
+# (the "License"); you may not use this file except in compliance with
+# the License.  You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+-->
+
+## 描述
+
+`ldap-auth` 插件可用于给路由或服务添加 LDAP 身份认证，该插件使用 [lualdap](https://lualdap.github.io/lualdap/) 连接 LDAP 服务器。
```

Comment:
```suggestion
`ldap-auth` 插件可用于给路由（Route）或服务（Service）添加 LDAP 身份认证，该插件使用 [lualdap](https://lualdap.github.io/lualdap/) 连接 LDAP 服务器。
```

Reply:
I think we could present the Chinese definitions of these terms in the Chinese documents, and I will check them in other documents

Reply:
ok